src/fsharp/pars.fsy

```diff
@@ -2403,8 +2409,11 @@ hardwhiteLetBindings:
         BindingSetPreAttrs(mLetKwd,isRec,isUse,
           (fun attrs vis -> 
             let binds = localBindingsBuilder attrs vis mLetKwd 
-            if not isRec && List.length binds > 1 then 
-                    reportParseErrorAt mLetKwd (FSComp.SR.parsLetAndForNonRecBindings()) 
+            if isRec then [],binds else
```

Comment:
I'd prefer a new list function ``List.isNilOrSingleton`` so

    if not isRec && not (List.isNilOrSingleton binds) then ...